scripts/building/run_coverage_tests.py

```diff
@@ -96,4 +113,15 @@
         pytest_command = pytest_command + ["--model-name", args.model_name]
 
     # pytest --junit-xml=test-results.xml --cov=azure.ai.ml --cov-report=html --cov-report=xml -ra ./tests/*/unittests/
-    run_command(pytest_command)
+    error_code, _ = run_command(pytest_command, throw_on_retcode=False)
+    if error_code == 1:
```

Comment:
maybe add some comments on where we can know the return code?

Reply:
OK